ticketmaster/aurora
# feat(Breakpoints): Update breakpoints to match industry standards

**What**: I am increasing breakpoint min-widths by 1px; decrease breakpoint max-widths by 1px.



**Why**: I am doing this to match industry standards.



**How**: I am increasing breakpoint min-widths by 1px; decrease breakpoint max-widths by 1px.



**How to test**:
1. Fetch and check out this branch.
2. Symlink the `aurora` repo's root directory with your web application by executing `npm link $path_to_repo` from within your web application's repo's root directory. Alternatively, execute `yarn run build` within the `aurora` repo's root directory and then copy the resulting `dist` files over into your web applications `aurora` `node_modules` directory.
3. Run your web application and view the impact of the updated breakpoints

**Checklist**:
